<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1115</width>
    <height>543</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1115</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fairy Batle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>181</width>
      <height>122</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{border: 2px solid;border-radius:8px;}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>9</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>35</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>35</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>61</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRegist">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogout">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{border: 2px solid;border-radius:8px;}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonRefreshPlayer">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>玩家：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonInfoPlayer">
       <property name="text">
        <string>查看更多</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidgetPlayer">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>70</number>
       </attribute>
       <column>
        <property name="text">
         <string>玩家名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>在线状态</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>胜率</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_3
{
border: 2px solid;
border-radius:8px;
}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>决斗场：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidgetArena">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>55</number>
       </attribute>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>等级</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>生命值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>攻击力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>防御力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>生命力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>敏捷力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>闪避率</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>暴击率</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonRefreshArena">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>查看决斗场</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_4{border: 2px solid;border-radius:8px;}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonGiveOut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>送出</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonHeal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>补充生命值</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonRefreshFairy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>查看我拥有的精灵</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelFairy">
       <property name="text">
        <string>当前拥有的精灵：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidgetFairy">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>10</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>55</number>
       </attribute>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>特长</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>等级</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>生命值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>攻击力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>防御力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>生命力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>敏捷力</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>闪避率</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>暴击率</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_5{border: 2px solid;border-radius:8px;}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButtonDeath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>决斗赛</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonAttack">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>攻击</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonLevel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>升级赛</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QPlainTextEdit" name="plainTextEditState">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_6{border: 2px solid;border-radius:8px;}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>服务器IP地址：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditServer">
       <property name="text">
        <string>192.168.132.1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>140</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_7{border: 2px solid;border-radius:8px;}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>宠物徽章：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listWidgetBadage"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonRefreshBadage">
       <property name="text">
        <string>刷新徽章列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
